test/e2e/autoscale.go

```diff
@@ -308,11 +308,11 @@ func checkPodScale(ctx *testContext, targetPods, minPods, maxPods float64, durat
 				return errors.New("interim scale didn't fulfill constraints: " + mes)
 			}
 			// A quick test succeeds when the number of pods scales up to `targetPods`
-			// (and, for sanity check, no more than `maxPods`).
+			// (and no more than `maxPods`).
```

Comment:
tempted to suggest something like:

```suggestion
			// (and, as an extra check, no more than `maxPods`).
```

just to avoid losing the gist the previous wording was trying to add. Maybe actually the brackets are already doing that job, though.

Reply:
Added.